tensorflow_addons/activations/gelu.py

```diff
@@ -14,13 +14,16 @@
 # ==============================================================================
 
 import tensorflow as tf
+from typeguard import typechecked
```

Comment:
unused import?

Reply:
It reminds me that I need to re-enable this check with flake8